README.md

```diff
@@ -13,9 +13,14 @@ You will need a basic understanding of how to use the command line on your compu
 
 You will need Node and npm installed on your computer. Node is a JavaScript runtime used for developer tooling, and npm is the package manager included with Node. If you have a package manager installed for your operating system, setup can be as straightforward as:
 
-* macOS: `brew install node`
-* Windows: `choco install nodejs`
-* Ubuntu: `apt install nodejs npm`
+* macOS: `brew install node@14`
+* Windows: `choco install nodejs --version=14.20.1`
+* Ubuntu: 
+	* `sudo apt install curl`
+	* `curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -`
+	* `sudo apt install nodejs`
+
+**Note:** WordPress currently only supports up to NodeJS 14.x and npm < 7.
```

Comment:
May be a slight change like this in the note can be a little more readable👇: 

`**Note:** WordPress currently only supports NodeJS version up to 14.x and npm  version up to 7`

Reply:
I think I prefer the way that it is in the pull request currently.